package.json

```diff
@@ -30,60 +30,61 @@
   },
   "dependencies": {
     "@nestjs/cache-manager": "^3.0.1",
-    "@nestjs/common": "^10.0.0",
-    "@nestjs/config": "^3.1.1",
-    "@nestjs/core": "^10.0.0",
-    "@nestjs/platform-express": "^10.4.5",
-    "@nestjs/serve-static": "^4.0.0",
-    "cache-manager": "^6.4.1",
+    "@nestjs/common": "^11.0.20",
+    "@nestjs/config": "^4.0.2",
+    "@nestjs/core": "^11.0.20",
+    "@nestjs/platform-express": "^11.0.20",
+    "@nestjs/serve-static": "^5.0.3",
+    "cache-manager": "^6.4.2",
     "drizzle-orm": "^0.36.1",
     "immer": "^10.1.1",
-    "joi": "^17.11.0",
-    "pg": "^8.11.3",
+    "joi": "^17.13.3",
+    "pg": "^8.14.1",
+    "qs": "^6.14.0",
```

Comment:
Has our application been directly importing `qs`? Should it be listed as a direct dependency?

Reply:
I'll remove this
I was trying it to resolve some failures.